### PR TITLE
Add generic repeater loop.

### PR DESCRIPTION
**Pull request by robbintt**
Great inline in-game replacement for one offs.

### CODE REVIEW

File: repeat.lic [hunk 1]
```diff
@@ -0,0 +1,14 @@
+=begin
+  Documentation:
```
Comment (robbintt) [inline repeat.lic:2]: the repeated items must be double quoted e.g. `;repeat "get potato from my backpack" "put potato in urn"`

**Comment by Hiinky** [comment]
This already exits.  ;repos download multi.  Accepts scripts also
syntax:  ;multi 3,do thing one, do thing two,;script 2

**Comment by robbintt** [comment]
> This already exists. ;repos download multi. Accepts scripts also
> syntax: ;multi 3,do thing one, do thing two,;script 2

Thanks, where does that exist under source control?

**Comment by jandersson** [comment]
Hello! Looks good to me. With respect to the duplicate concept, I don't see any issue with this repo maintaining its own version of it. If the original author wants to submit `multi` then it can be taken from there.

File: repeat.lic (hunk 1 above)
Comment (robbintt) [reply repeat.lic:2]: will add docs in separate PR

**Pull request merged**